CONTRIBUTING.md

```diff
@@ -63,6 +67,39 @@ $ git tag -a v{version}
 $ git push --tags
 ```
 
+Testing with tox
+================
+The `tox` tool automatically creates appropriately configured Python
+virtual environments that can be used to run the code quality and
+verification tests, avoiding the need for the user to create an
+appropriate Python virtual environment.
+
+The `tox.ini` settings should work with versions of tox >= 2.9.1; the
+latest tox version available for your system's Python version should
+be available in your virtualenv if you installed the developer tools
+using `pip install -e .[dev]`.
+
+If you want to run the recommended code quality and verification tests
+you can use the `tox` command with no arguments:
+
+```shell
+$ tox
+```
+
+*NOTE:* By default tox is configured to run the pytest coverage tests for
+Python versions 3.6 to 3.12, inclusively, skipping any versions that aren't
```

Comment:
Not sure we need or want to hardcode python versions here. It will get out-of-date very quickly and likely will be forgotten.

Reply:
I can change that to Python versions 3.6 and later if that is preferred.

Reply:
I've updated the verbiage to be agnostic of Python versions.